micro_sam/training/util.py

```diff
@@ -43,8 +43,8 @@ def get_trainable_sam_model(
     checkpoint_path: Optional[Union[str, os.PathLike]] = None,
     freeze: Optional[List[str]] = None,
     return_state: bool = False,
-    use_lora: bool = False,
-    rank: Optional[int] = None,
+    lora_rank: Optional[int] = None,
+    lora_kwargs: Optional[Dict] = None,
```

Comment:
I removed the `use_lora`, since it is redundant with `rank = None` and renamed `rank` -> `lora_rank` to have a more explicit name.
I added `lora_kwargs`, so that we can also customize the PEFT SAM layers from here.

Reply:
Looks good to me. Thanks!

Reply:
For me as well :handshake: